CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/PluginParametersSwingDialog.java

```diff
@@ -174,6 +231,38 @@ public String getResult() {
         return result;
     }
 
+    /**
+     * Generates the text of the PluginParameterPane acceptance button based on keywords in the Pane title.
+     * This implementation is useful as it does not require the convoluted work of enabling developers to set 
+     * the acceptance button text at the plugin creation level.
+     * 
+     * @param title
+     * @return 
+     */
+    private String getAcceptanceButton() {
+        for (final String keyWord : acceptanceKeyWords){
+            if (StringUtils.containsIgnoreCase(title, keyWord)){
+                return keyWord;
+            }
+        }
+        return "OK";
+    }
+    
+    /**
+     * Checks to see if the acceptance button was selected.
+     * 
+     * @param result
+     * @return 
+     */
+    public static boolean isAccepted(final String result) {
+        for (final String keyWord : acceptanceKeyWords){
```

Comment:
wonder if this function could benefit from only checking `result`? That way it only needs to check one value (which should hopefully contain the correct value) rather than re-iterating through all the possible options again to find it.

Reply:
The object attribute result and the parameter result are essentially the same reference (silly implementation on my part). As the result could be "Cancel" evaluating the attribute against the parameter would return true when we want false. I've has a look at the implementation and decided on the following change: Instead of returning the string value of the button as the result when clicking the acceptance button, we return "OK" no matter the label of the button. That way we only need to check that the result is "OK" to verify if the button was pressed.